src/domains/citizen-auth-common/99_variables.tf

```diff
@@ -111,3 +111,13 @@ variable "cidr_subnet_redis_common_itn" {
   type        = list(string)
   description = "Redis Common address space."
 }
+
+variable "subnet_pendpoints_id" {
+  type        = string
+  description = "Id of the subnet which holds private endpoints"
+}
+
+variable "resource_group_common" {
+  type        = string
+  description = "Name of the common resource group"
+}
```

Comment:
These variables can be removed since we are getting this information from the common_values module
```suggestion

```